docs/pages/tutorials/github-oauth/sveltekit.md

```diff
@@ -100,9 +104,10 @@ Create an API route in `routes/login/github/+server.ts`. Generate a new state, c
 
 ```ts
 // routes/login/github/+server.ts
-import { github } from "$lib/server/auth";
-import { generateState } from "arctic";
 import { redirect } from "@sveltejs/kit";
+import { generateState } from "arctic";
+
+import { github } from "$lib/server/auth";
```

Comment:
Can you remove the whitespace between imports (except for ones between regular and type imports)?

E.g.

```
port { redirect } from "@sveltejs/kit";
import { generateState } from "arctic";
import { github } from "$lib/server/auth";

import type { RequestEvent } from "@sveltejs/kit";
```

Reply:
Yep, even though I think it's easier to understand which imports are pkg dependencies and which imports are from your codebase. (Type imports are an exception)